pkg/service/group_service_factory.go

```diff
@@ -20,7 +22,7 @@ type NsGroupService struct {
 	chapUserService                   *ChapUserService
 	masterKeyService                  *MasterKeyService
 	alarmService                      *AlarmService
-	volumeService                     *VolumeService
+	volumeService                     servicefactory.VolumeServiceHelper
```

Comment:
This change will break the SDK since you are replacing volumService with servicefactory.VolumeServiceHelper .

Reply:
This is just a sample. Was thinking of creating an interface for all services and using that here.  Is there an automated way to do that?